about/team/styles.css

```diff
@@ -0,0 +1,9 @@
+figcaption {
+  text-align: center;
+}
```

Comment:
Do we want to center all captions everywhere?

Reply:
seedcase-project/seedcase-theme#27